### PR TITLE
markdownlint changes

### LINKED ISSUE #17: markdownlint not being enforced
This page uses markdownlint, but there are 26 errors listed with `markdownlint-cli2 "**/*.md"`. I don't have the energy to fix them right now, but this is too irritating for my linter obsessed brain to pass up.

Worth considering a GitHub action that automatically lints, like this one:

```yaml
on: [push, pull_request]

jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: nosborn/github-action-markdown-cli@v3.2.0
        with:
          files: .
```

### PR DESCRIPTION
**Pull request by NotNite**
Closes #17 (I lied, bitch, I DO have energy).

- Edits the Markdown files to comply with our markdownlint rules (almost all do not impact visuals of wiki).
- Adds a new markdownlint rule exception.
- Sets up a GitHub Action to lint Markdown for us.

### CODE REVIEW

**Comment by NotNite** [comment]
```
$ markdownlint-cli2 "**/*.md"
markdownlint-cli2 v0.5.1 (markdownlint v0.26.2)
Finding: **/*.md
Linting: 46 file(s)
Summary: 0 error(s)
```

**Review by initmd — APPROVED** [review]

File: src/wiki/development/build-instructions.md [hunk 1]
```diff
@@ -123,15 +123,15 @@ flatpak-builder --user --install flatbuild org.prismlauncher.PrismLauncher.yml
 2. Choose a place to install Qt.
 3. Choose the components that you wish install.
 
-   - You need Qt 5.12.x 64-bit ticked. (or a newer version)
-      - Alternatively you can choose Qt 6.0 or newer
-   - You need Tools/Qt Creator ticked.
-   - Other components are selected by default, you can un-tick them if you don't need them.
+    - You need Qt 5.12.x 64-bit ticked. (or a newer version)
+    - Alternatively you can choose Qt 6.0 or newer
```
Comment (TayouVR) [inline src/wiki/development/build-instructions.md:127]: the extra indentation here got removed
```
        - Alternatively you can choose Qt 6.0 or newer
```
its already merged now, so guess needs a new pr

Comment (NotNite) [reply src/wiki/development/build-instructions.md:127]: Oh, whoops.

File: src/wiki/getting-started/installing-optifine.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ eleventyNavigation:
 
 Because OptiFine is an old, closed-source project, it has evolved to cause many mod incompatibilities, and doesn't do as good of a job as some the alternatives out there.
 
-Also, if you're using the Fabric mod loader, there's no native support for it, and the current [workaround](#fabric) is known to be buggy and quite unstable, due to the very nature of OptiFine and OptiFabric.
+Also, if you're using the Fabric mod loader, there's no native support for it, and the current workaround is known to be buggy and quite unstable, due to the very nature of OptiFine and OptiFabric.
```
Comment (TayouVR) [inline src/wiki/getting-started/installing-optifine.md:11]: there are a couple links in this area, which got removed... is that intentional?

Comment (NotNite) [reply src/wiki/getting-started/installing-optifine.md:11]: Those links didn't resolve when I tested it locally, and they were erroring in markdownlint. I'm not sure what else I was supposed to do with them.

Comment (TayouVR) [reply src/wiki/getting-started/installing-optifine.md:11]: they are supposed to go to a header on the same page, but idk... maybe they were always fucked

Comment (TayouVR) [reply src/wiki/getting-started/installing-optifine.md:11]: they were working on the polymc.org deployment, so it should work here too... 🤔

**Pull request merged**